etc/grub/grub.cfg.ww

```diff
@@ -120,7 +120,7 @@ menuentry "Single-stage boot (no compression)" --id single-stage-nocompress {
     boot
 }
 
-menuentry "Two stage boot with dracut" --id dracut {
+menuentry "Two stage boot with dracut {{ if eq .Root "persistent"}}(persistent){{ end }}" --id dracut {
```

Comment:
Don't make the menuentry dynamic: make a separate menuentry that is selected when persistent provisioning is desired.